fnl/nfnl/fs.fnl

```diff
@@ -93,6 +93,15 @@
       (replace-extension "lua")
       (replace-dirs "fnl" "lua")))
 
+(fn standardize-path [path]
+  "Replaces all non standard path separators with the standard forward slash"
+  (str.replace path "\\" "/"))
+
+(fn correct-separators [path]
+  "Replaces all path separators with the ones appropriate for this system"
+  (str.replace path "\\" (path-sep))
+  (str.replace path "/" (path-sep)))
```

Comment:
I'm not sure of the right answer yet, but something about how these two functions work makes me think there will be issues 🤔  also I didn't think string.gsub mutated the original string? So only the second `str.replace` actually does things here right?

My thinking is along the lines of "you can have a `\` inside a unix path", and I guess you can have a `/` in a Windows path. So we want to make sure those stay intact when messing with paths. The Conjure code for this probably falls into the same trap, I'll have to check.

Reply:
Ugh. This is just my not being so familiar with fennel. Sounds like its better to use vim.fs.normalize than my changes